src/Symfony/Bridge/PhpUnit/Tests/CoverageListenerTest.php

```diff
@@ -16,6 +16,11 @@ public function test()
             $this->markTestSkipped('This test cannot be run on HHVM.');
         }
 
+        exec('php --ri xdebug -d zend_extension=xdebug.so 2> /dev/null', $output, $returnCode);
```

Comment:
I have two quick questions:

* Would that command work on Windows?
* Would that command work in PHP 7.2, where extensions can remove the `.so` and `.dll` suffix? (https://wiki.php.net/rfc/load-ext-by-name)

Reply:
> Would that command work on Windows?

No it does not work on windows. But we don't care as this test is skipped on windows ;)

> Would that command work in PHP 7.2, where extensions can remove the .so and .dll suffix? (https://wiki.php.net/rfc/load-ext-by-name)

I don't know but it should. If it does not work it means PHP has introduced a major BC break.